design-principles.md

```diff
@@ -8,14 +8,65 @@ permalink: /design-principles/
 * TOC Placeholder
 {:toc}
 
-## Introduction
+Introduction
+------------
 
 Flutter is written based on some core principles that were mostly
 intuited from past experiences with other platforms such as the Web
 and Android, some of which are summarised below.
 
 This document is most useful if you want to contribute to Flutter, as
-then we hope you will also follow these philosophies.
+then we hope you will also follow these philosophies. Please also read
+our
+[style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo)
+for more specific guidelines regarding writing Dart code for Flutter.
+
+
+Code review and checking in code
+--------------------------------
+
+Every PR must be code-reviewed before check-in, including things like
```

Comment:
I might talk a bit about the purpose of code review, or link to something that does.  For me, a big part of it is sharing knowledge.  That's why even the most experienced engineers get their patches reviewed, so that through the process they are sharing the knowledge of their patch with someone else.  Then always 2 people understand some system instead of one.  Obviously there is also the correctness validation, but to me that's secondary to the bi-directional information sharing/teaching.